Eastern-Research-Group/expert-query
# Feature/91 add glossary to client

## Related Issues:
* [EQ-91](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-91)

## Main Changes:
* Added the [glossary component](https://github.com/Eastern-Research-Group/glossary) to the client-side UI.
* Added two _eslint_-related dev dependencies to the server.
  * The two plugins were specified in the server-side config, but not yet installed.

## Steps To Test:
1. Navigate to the application's home page: [localhost:3000/attains](localhost:3000/attains).
2. Click the **Glossary** button and confirm the glossary slides open from the right.
3. Alternatively, a test glossary link has been added to the "Current Query" label. When clicked, this should link to the "Acidity" glossary term.
